changes/pr314.yaml

```diff
@@ -0,0 +1,23 @@
+# An example changelog entry
+#
+# 1. Choose one (or more if a PR encompasses multiple changes) of the following headers:
+#   - feature
+#   - enhancement
+#   - fix
+#   - deprecation
+#   - breaking (for breaking changes)
+#   - migration (for database migrations)
+#
+# 2. Fill in one (or more) bullet points under the heading, describing the change.
+#    Markdown syntax may be used.
+#
+# 3. If you would like to be credited as helping with this release, add a
+#    contributor section with your name and github username.
+#
+# Here's an example of a PR that adds an enhancement
+
+enhancement:
+  - "Add service role annotations - [#314](https://github.com/PrefectHQ/server/pull/314)"
```

Comment:
```suggestion
  - "Helm: Allow annotations to be provided for service accounts- [#314](https://github.com/PrefectHQ/server/pull/314)"
```

Reply:
Absolutely - I wasn't sure if there was a convention you followed for that sort of categorisation, but that definitely makes it clearer